### PR TITLE
B 20

### LINKED ISSUE #20: Documentation Various issues and old goca
Under HTML doc there are still:
* Formatting issues (markdown or typos)
* Deprecated Terraform syntax (before 0.12) examples

### PR DESCRIPTION
**Pull request by jaypif**
fix #20

### CODE REVIEW

File: website/docs/index.html.markdown [hunk 1]
```diff
@@ -33,23 +33,22 @@ variable "one_username" {}
...
-  endpoint = "${var.one_endpoint}"
-  username = "${var.one_username}"
-  password = "${var.one_password}"
+  endpoint = var.one_endpoint
+  username = var.one_username
+  password = var.one_password
 }
 
-# Create a new group of users to the OpenNebula cluster
+# Examepl Create a new group of users to the OpenNebula cluster
```
Comment (AlexVerchere) [inline website/docs/index.html.markdown]: Here's a typo: 'Example'

**Comment by AlexVerchere** [comment]
Everything looks good, but you can also re-indent the `docs/r/virtual_machine_group.html`

File: website/docs/r/group.html.markdown [hunk 2]
```diff
@@ -76,16 +75,16 @@ The following arguments are supported:
...
 `quotas` supports the following arguments:
 
-* `datastore_quotas` - (Optional) List of datastore quotas. See [Datastore quotas parameters](#datastore-quotas) below for details.
-* `network_quotas` - (Optional) List of network quotas. See [Network quotas parameters](#network-quotas) below for details.
-* `image_quotas` - (Optional) List of image quotas. See [Image quotas parameters](#image-quotas) below for details
-* `vm_quotas` - (Optional) See [Virtual Machine quotas parameters](#vm-quotas) below for details
+* `datastore_quotas` - (Optional) List of datastore quotas. See [Datastore quotas parameters](#datastore-quotas-parameters) below for details.
+* `network_quotas` - (Optional) List of network quotas. See [Network quotas parameters](#network-quotas-parameters) below for details.
+* `image_quotas` - (Optional) List of image quotas. See [Image quotas parameters](#image-quotas-parameters) below for details
+* `vm_quotas` - (Optional) See [Virtual Machine quotas parameters](#virtual-machine-quotas-paramters) below for details
```
Comment (treywelsh) [inline website/docs/r/group.html.markdown]: typo, broken link

File: website/docs/r/virtual_machine.html.markdown [hunk 3]
```diff
@@ -74,20 +74,20 @@ resource "opennebula_virtual_machine" "demo" {
...
-* `os` - (Optional) See [OS parameters](#os-vm) below for details.
-* `disk` - (Optional) Can be specified multiple times to attach several disks. See [Disks parameters](#disks-vm) below for details.
-* `nic` - (Optional) Can be specified multiple times to attach several NICs. See [Nic parameters](#nic-vm) below for details.
-* `vmgroup` - (Optional) See [VM group parameters](#os-vmg) below for details. Changing this argument triggers a new resource.
+* `memory` - (Optional) Amount of RAM assigned to the VM in MB. **Mandatory if** `template_id` **is not set**.
+* `context` - (Optional) Array of free form key=value pairs, rendered and added to the CONTEXT variables for the VM. Recommended to include: `NETWORK = "YES"` and `SET_HOSTNAME = "$NAME"`. If a `template_id` is set, see [Instantiate from a template](#instantiate-from-a-template) for details.
+* `graphics` - (Optional) See [Graphics parameters](#graphics-parameters) below for details.
+* `os` - (Optional) See [OS parameters](#os-parameters) below for details.
+* `disk` - (Optional) Can be specified multiple times to attach several disks. See [Disks parameters](#disks-parameters) below for details.
```
Comment (treywelsh) [inline website/docs/r/virtual_machine.html.markdown]: typo, broken link (the disk)

File: website/docs/r/group.html.markdown (hunk 2 above)
Comment (jaypif) [reply website/docs/r/group.html.markdown]: indeed thanks

File: website/docs/r/virtual_machine.html.markdown (hunk 3 above)
Comment (jaypif) [reply website/docs/r/virtual_machine.html.markdown]: indeed

**Review by treywelsh — APPROVED** [review]
Reviewed with @AlexVerchere
It's OK

**Pull request merged**